docs/standard/schema.core.rst

```diff
@@ -64,6 +64,18 @@ See :ref:`forks-and-variants` for a complete description of what
 is a software variant and how to handle forked software as a parser or
 an author.
 
+Key ``components``
+~~~~~~~~~~~
+
+-  Type: array of strings (URLs)
+-  Presence: optional
+
+Some software projects are organized in more than one repository.
+In addition to the key ``url``, this key gives you an option to specify
+more than one URL to further relevant repositories. If you use the key,
```

Comment:
I would spend a few words here to specify what a "relevant repository" is. I mean, can they be random repos spread all over the Internet or they have to respect some sort of pattern, like e.g., `/sameUrlOrg/repo1`, `/sameUrlOrg/repo2` etc?

Reply:
Agreed on expanding what we expect this key to be used for, with a couple of examples.

IMO we should state that `components` should be parts of the same project / sofware, but not related software in general. So, for example, the frontend, backend and apps of Google Maps can be `components` in the Google Maps `publiccode.yml`, but Google Assistant should not.

I think they should point wherever the components are, even on other orgs or forges. Typically that should not be the case, but I don't see a reason to not allow it.

Should we also explicitly mention guidance about plugins? Should they be `components`? Probably not if they're not essential to run the software.

WDYT @Smart-City-Muenster, @libremente?

Reply:
Hi there,

I've just [pushed a small addition](https://github.com/publiccodeyml/publiccode.yml/pull/169/commits/7ab2b9ecdf731ab5f4242a01c9f4d123312fd1a7) that specifies what repositories should go into "components".

I was reluctant to explicitly mention plugins, because I do not necessarily see harm in seeing a plugins-repo in this field, although I also do not want to encourage people to add it there ;)